pyls/plugins/jedi_completion.py

```diff
@@ -50,8 +51,25 @@ def pyls_completions(config, document, position):
 
     settings = config.plugin_settings('jedi_completion', document_path=document.path)
     should_include_params = settings.get('include_params')
+    include_params = (use_snippets(document, position) and
+                      snippet_support and should_include_params)
+    return [_format_completion(d, include_params) for d in definitions] or None
 
-    return [_format_completion(d, snippet_support and should_include_params) for d in definitions] or None
+
+def use_snippets(document, position):
+    """
+    Determine if returning snippets in code completions.
+
+    Returns False if the completion is running on an import statement
+    Returns True otherwise
```

Comment:
Let's change this docstring to be

    Determine if it's necessary to return snippets in code completions.

    This returns `False` if a completion is being requested on an import statement,
    `True` otherwise.